cfgov/cfgov/settings/base.py

```diff
@@ -501,3 +501,8 @@
 
 # Akamai
 ENABLE_AKAMAI_CACHE_PURGE = os.environ.get('ENABLE_AKAMAI_CACHE_PURGE', False)
+if ENABLE_AKAMAI_CACHE_PURGE:
+    AKAMAI_USER = os.environ['AKAMAI_USER']
```

Comment:
It's reasonable to assume the Akamai vars will be available if `ENABLE_AKAMAI_CACHE_PURGE` is set, but calling os.environ values directly risks a KeyError that would halt the whole show. 
Could we use `os.environ.get()` for each sub-value to scope the risk to just the purge process?